temba/settings_common.py

```diff
@@ -351,11 +351,11 @@
         "welcome_packs": [dict(size=5000, name="Demo Account"), dict(size=100000, name="UNICEF Account")],
         "title": _("Visually build nationally scalable mobile applications"),
         "description": _("Visually build nationally scalable mobile applications from anywhere in the world."),
-        "credits": _("Copyright &copy; 2012-2017 UNICEF, Nyaruka. All Rights Reserved."),
```

Comment:
tsk, tsk.

Reply:
Not only is removing our copyright inappropriate, your rapidpro fork should be public to comply with AGPL.

Reply:
Sorry about this PR it was pushed by mistake